docs/source/developer_guide/pipelines.md

```diff
@@ -125,7 +125,7 @@ For more details on the **pipeline json definition** see it's [json schema](http
 				"label": "overview",
 				"component_parameters": {
 					"filename": "demo-pipelines/overview.ipynb",
-					"runtime_image": "elyra/tensorflow:1.15.2-py3",
+					"runtime_image": "tensorflow/tensorflow:7c01f75d58fadc2cd1109d5baac1925ed131e05925d840b1b49363c794d1c4db",
```

Comment:
`[elyra/tensorflow](https://hub.docker.com/r/elyra/tensorflow/tags)` or tensorflow/tensorflow for that part of the pipeline def?